DataFormats/StdDictionaries/src/classes_def_map.xml

```diff
@@ -16,6 +16,7 @@
  <class name="std::map<std::basic_string<char>,std::vector<std::pair<std::basic_string<char>,double> > >"/>
  <class name="std::map<std::basic_string<char>,unsigned int>"/>
  <class name="std::map<std::basic_string<char>,unsigned long>"/>
+ <class name="std::map<std::basic_string<char>,unsigned __int64>"/>
```

Comment:
So the `unsigned long` above is not sufficient?

Reply:
So indeed, this PR does not appear to fix the underlying problem. I'll close it until we understand better..